app/utils/githubHelpers.js

```diff
@@ -0,0 +1,59 @@
+import axios from 'axios'
+
+var id = "YOUR_CLIENT_ID";
+var sec = "YOUR_SECRET_ID";
+var param = `?client_id=${id}&client_secret=${sec}`;
+
+function getUserInfo (username = 'bartekkustra') {
+  return axios.get(`https://api.github.com/users/${username + param}`);
+}
+
+function getRepos (username = 'bartekkustra') {
+  return axios.get(`https://api.github.com/users/${username}/repos${param}&per_page=100`);
+}
+
+function getTotalStars (repos) {
+  return repos.data.reduce((prev, current) => {
+    return prev + current.stargazers_count
+  }, 0)
+}
+
+async function getPlayersData (player) {
+  try {
+    const repos = await getRepos(player.login)
+    const totalStars = await getTotalStars(repos)
+    return {
+      followers: player.followers,
```

Comment:
This would be a bit cleaner like this:

```
      const {followers, login} = player;
      const repos = await getRepos(login)
      const totalStars = await getTotalStars(repos)

      return {
        followers,
        totalStars
      }
```